stable/cockroachdb/README.md

```diff
@@ -53,7 +53,21 @@ helm install --name my-release stable/cockroachdb
 
 Note that for a production cluster, you are very likely to want to modify the `storage.persistentVolume.size` and `storage.persistentVolume.storageClass` parameters. This chart defaults to `100 GiB` of disk space per Pod, but you may want more or less depending on your use case, and the default PersistentVolume `storageClass` in your environment may not be what you want for a database (e.g. on GCE and Azure the default is not SSD).
 
-If you are running in secure mode (with configuration parameter `tls.enabled` set to `yes`/`true`), then you will have to manually approve the cluster's security certificates as the Pods are created. You can see the pending CSRs (certificate-signing requests) by running `kubectl get csr`, and approve them by running `kubectl certificate approve <csr-name>`. You'll have to approve one certificate for each Node (e.g. `default.node.eerie-horse-cockroachdb-0` and one client certificate for the Job that initializes the cluster (e.g. `default.node.root`).
+If you are running in secure mode (with configuration parameter `tls.enabled` set to `yes`/`true`) and `tls.certs.provided` is set to `no`/`false`, then you will have to manually approve the cluster's security certificates as the Pods are created. You can see the pending CSRs (certificate-signing requests) by running `kubectl get csr`, and approve them by running `kubectl certificate approve <csr-name>`. You'll have to approve one certificate for each Node (e.g. `default.node.eerie-horse-cockroachdb-0` and one client certificate for the Job that initializes the cluster (e.g. `default.node.root`).
+For "Bring your own certs" scenario, please set `tls.certs.provided` is set to `yes`/`true` and create secrets `tls.certs.clientRootSecret` and `tls.certs.nodeSecret` with the following content
```

Comment:
Thank you.  I adopted your wording. To say the true, you don't need cockroachdb binary to generate certs, I used Vault/Terraform PKI in my case

Reply:
True, we also use Vault to generate certs for some clusters. I suggested using the cockroachdb binary to generate certs because it's a simple example of how to do it.

Reply:
agree